cinema/management/commands/wait_for_db.py

```diff
@@ -0,0 +1,16 @@
+import time
+
+from django.core.management.base import BaseCommand
+from django.db import connections, OperationalError
+
+
+class Command(BaseCommand):
+
+    def handle(self, *args, **kwargs):
+        while True:
+            try:
+                connections["default"].ensure_connection()
+                break
+            except OperationalError:
+                self.stdout.write("Server loading wait...")
+                time.sleep(1)
```

Comment:
Consider adding a maximum retry limit or timeout to prevent the script from running indefinitely if the database is not available. This can help avoid potential issues in production environments.